test/db/abi/compilers/gcc

```diff
@@ -26,7 +26,7 @@ CMDS=<<EOF
 id~?
 EOF
 EXPECT=<<EOF
-25813
+24358
```

Comment:
What made this significant difference? Can you show an excerpt of what disappeared?

Reply:
Mainly because `debug_ranges` `debug_loc` sections were changed to load by need, and `debug_ranges` was not used, so the dump of `debug_ranges` was missing.

Moreover, these sections need the information in the `debug_info` `CompUnit` to be loaded correctly. In the past I used it the wrong way, and this is fixed in this PR.

Reply:
the diff

[diff.txt](https://github.com/rizinorg/rizin/files/12597166/diff.txt)